Few fixes

sparksam opened the pull request:
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

david-oc-miller commented on vagrant/provisioning/roles/arkcase-cognito/tasks/main.yml:
```diff
@@ -21,4 +21,12 @@
\ No newline at end of file
+    backup: yes
+
+- name: setup configuration file
```
Is this setup also done in the task above this one, which calls the ../arkcase-app/templates/arkcase-portal-cognito-server.yaml template?

sparksam replied:
No, the database settings are not done in that file. I wonder if it wouldn't be ideal to merge both files and keep them in the configuration folder.

david-oc-miller replied:
Merging the files would be great if you have time to do it.  Good catch.

david-oc-miller commented on vagrant/provisioning/roles/arkcase-ldap-authority/templates/server.yml:
```diff
@@ -1,22 +1 @@
-  file:
-    name: {{ root_folder }}/log/arkcase-ldap-authority/arkcase-ldap-authority.log
\ No newline at end of file
+spring.data.mongodb.uri: "mongodb://{{ microservice_mongodb_username }}:{{ microservice_mongodb_password }}@{{ microservice_mongodb_host | default(internal_host) }}:{{ mongodb_port | default(27017) }}/{{ microservice_mongodb_database }}"
```
Since we're making some improvements, can you also add a fact e.g. `mongodb_ssl` fact, by default set to `false`, and add `?ssl={{ mongodb_ssl | default (false) }}` at the end of the MongoDB URIs?

sparksam replied:
On it.

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup

Commit:
Few fixes
- Update MongoDB configuration
- Arkcase integration user for filemanagement
- Code Cleanup
